devlib/platform/gem5.py

```diff
@@ -127,6 +129,11 @@ def _start_interaction_gem5(self):
             # telnet connection.
             self.stderr_filename = f
 
+            # Enable power modelling if required
+            if self.power_models_dir:
```

Comment:
> Maybe we can still have some lowest-common-denominator configuration, e.g. the model could take power_model_args and power_model_env_vars parameters.

Yes, we could have at least that. However, if something else is needed to activate power models for a specific platform, `power_model_args` and `power_model_env_vars` will not be enough and it will be _implicitly_ assumed that the user as configured the external bits correctly anyway. 

Actually, by exposing power-related parameters, we will give the impression that devlib is setting everything up and running for power while it will actually setup things only partially for some platforms, and that can be misleading ...
Maybe this is just too platform-dependent and we should leave users to manage that using `gem5args_args` in `Gem5SimulationPlatform`.